### PR TITLE
Invalidate cache for blocklist XML paths.

### PR DESCRIPTION
**Pull request by Natim**
Maybe this should be handled in the kinto-amo plugin also the kinto-amo plugin is not necessarily on the kinto-writer. Let's discuss that with @leplatrem

### CODE REVIEW

File: kinto_signer/updater.py [hunk 1]
```diff
@@ -399,6 +399,14 @@ def _update_source_attributes(self, request, **kwargs):
     def invalidate_cloudfront_cache(self, request, timestamp):
         collection_path = "/v1%s*" % self.destination_collection_uri
         monitor_path = "/v1/buckets/monitor/collections/changes*"
+
+        paths = [collection_path, monitor_path]
+
+        if self.destination_collection_uri.startswith('/buckets/blocklists/'):
+            paths.append('/v1/blocklist/*')
+        elif self.destination_collection_uri.startswith('/buckets/blocklists-preview/'):
+            paths.append('/v1/preview/*')
```
Comment (leplatrem) [inline kinto_signer/updater.py]: and `staging` ?

Comment (Natim) [reply kinto_signer/updater.py]: what about it?

Comment (Natim) [reply kinto_signer/updater.py]: Oh you mean invalidating the blocklist for staging? We don't have one because staging is not a public collection.

**Comment by Natim** [comment]
> The other radical approach would be invalidate everything /v1/*

Isn't it too risky?

**Comment by leplatrem** [comment]
As discussed we can still add a setting

**Comment by Natim** [comment]
> As discussed we can still add a setting

I tried adding a settings but it means invalidating everything all the time :/

**Pull request closed without merging**